src/UserInput.ts

```diff
@@ -0,0 +1,40 @@
+const readline = require("readline");
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+export function askWhichBoard(): Promise<string> {
+  return new Promise((resolve, reject) => {
+    rl.question("Type 1 for initial board or 2 for custom board ", resolve);
+  });
+}
+
+export function askUserBoardPath(): Promise<string> {
+  return new Promise((resolve, reject) => {
+    rl.question("Path ", resolve);
+  });
+}
+
+export async function askUserMove(): Promise<void> {
```

Comment:
a promise here is useless

Reply:
I don't understand, since I'm using awaits just above, I don't have to be in an aync function and return a promise?